templates/patterns/context_nav/pattern-context-nav.html.twig

```diff
@@ -0,0 +1,12 @@
+{#
+/**
+ * @file
+ * Link block.
+ */
+#}
+{% include '@ecl/generic-component-context-nav' with {
```

Comment:
The template should `@ecl/ec-component-context-nav` and don't forget to add it on ecl-builder.config.js